styling-and-themes/overview.md

```diff
@@ -85,38 +85,42 @@ Before you can add Telerik Theming to your app, make sure that it is configured
         <ResourceDictionary.MergedDictionaries>
             <ResourceDictionary Source="Resources/Styles/Colors.xaml" />
             <ResourceDictionary Source="Resources/Styles/Styles.xaml" />
-            <local:TelerikTheming />
+            <local:TelerikThemeResources />
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
 </Application.Resources>
 ```
 
-**4.** Set the `Telerik` theme and the desired swatch in the `App.xaml.cs` file. This example uses the `Purple` swatch:
+**4.** Choose a theme by setting the `Theme` (`enum` of type `TelerikTheme`) property to the desired theme in the `App.xaml`:
+
+```XAML
+<local:TelerikThemeResources Theme="TelerikPurple" />
+```
+
+or `AppTheme` (`enum` of type `TelerikTheme`) property to the `App.xaml.cs` file.
 
 ```C#
-var telerikTheming = Application.Current
-                    .Resources
-                    .MergedDictionaries
-                    .OfType<TelerikTheming>()
-                    .Single();
-telerikTheming.Theme = TelerikTheming.Themes
-                    .Single(t => t.Theme == "Telerik" && t.Swatch == "Purple");
+TelerikThemeResources.AppTheme = TelerikTheme.TelerikPurple;
 ```
 
 The steps above apply the `Telerik` theme with its `Purple` swatch to the Telerik .NET MAUI components used across the app.
 
-For example, if you have the following `RadToggleButton` control:
+The `TelerikTheme` `enum` has the following values: `Default`, `PlatformLight`, `PlatformDark`, `TelerikMain`, `TelerikMainDark`, `TelerikOceanBlue`, `TelerikOceanBlueDark`, `TelerikPurple`, `TelerikPurpleDark`, `TelerikTurquoise`, and `TelerikTurquoiseDark`.
```

Comment:
Consider removing this paragraph and adding the information to the table with the swatches on line 30. For example:

| Theme | Swatch | Color | `TelerikTheme` `enum` |
| ----- | ------ | ----- | ----- |
| Platform | Light | ![Telerik UI for .NET MAUI Platform Light Theme](images/platform-light.png) | PlatformLight |
| Platform | Dark | ![Telerik UI for .NET MAUI Platform Dark Theme](images/platform-dark.png) | PlatformDark |